internal/uvm/create_wcow.go

```diff
@@ -204,10 +155,138 @@ func CreateWCOW(ctx context.Context, opts *OptionsWCOW) (_ *UtilityVM, err error
 		}
 	}
 
-	uvm.scsiLocations[0][0] = &SCSIMount{
-		vm:       uvm,
-		HostPath: doc.VirtualMachine.Devices.Scsi["0"].Attachments["0"].Path,
-		refCount: 1,
+	return doc, nil
+}
+
+// CreateWCOW creates an HCS compute system representing a utility VM.
+// The HCS Compute system can either be created from scratch or can be cloned from a
+// template.
+//
+// WCOW Notes:
+//   - The scratch is always attached to SCSI 0:0
+//
+func CreateWCOW(ctx context.Context, opts *OptionsWCOW) (_ *UtilityVM, err error) {
```

Comment:
I think it would be helpful to separate this into `CreateWCOW` and `CloneWCOW`, and make `CloneWCOW` take the template config as an argument directly. This would be clearer than just using options. We can unify common pieces of UVM setup in a common function.

Reply:
I think I had suggested that he combine it into one function to make the caller's code clearer, but I'm fine with this if you think it's preferred.

Reply:
I think it's better separated, since they are pretty different use cases, but it's not a big deal right now.

Reply:
In the interest of time let us keep this as it is for now. I can refactor this later.